doc/source/ray-air/benchmarks.rst

```diff
@@ -3,6 +3,57 @@ AIR Benchmarks
 
 Below we document key performance benchmarks for common AIR tasks and workflows.
 
+Bulk Ingest
+-----------
+
+This task uses the DummyTrainer module to ingest 200GiB of synthetic data.
+
+We test out the performance across different cluster sizes.
+
+- `Bulk Ingest Script`_
+- `Bulk Ingest Cluster Configuration`_
+
+For this benchmark, we configured the nodes to have reasonable disk size and throughput to account for object spilling.
```

Comment:
maybe say what the # workers are for (they're for consumption right?)

Reply:
Nice. Should we add some known limitations section for some of these benchmarks? Such as: it gets much slower if data doesn't fit in memory.